### PR TITLE
[DOC] Update bert/index.rst

### PR DESCRIPTION
**Pull request by jamiekang**
KoBERT description added.

## Description ##
Korean BERT section added with a code snippet.

### Essentials ###
- [ ] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

cc @dmlc/gluon-nlp-team

### CODE REVIEW

File: scripts/bert/index.rst [hunk 1]
```diff
@@ -115,9 +117,19 @@ Finally, GluonNLP also supports the "`DistilBERT <https://arxiv.org/abs/1910.011
     words, valid_len = mx.nd.array([sample[0]]), mx.nd.array([sample[1]])
     seq_encoding, cls_encoding = model(words, valid_len);
 
+Finally, GluonNLP also suports Korean BERT pre-trained model, [KoBERT](https://github.com/SKTBrain/KoBERT), trained with Korean wiki dataset ('kobert_news_wiki_ko_cased').
+
+.. code-block:: python
+
+    !pip install -U https://github.com/dmlc/gluon-nlp/archive/v0.9.0.tar.gz
```
Comment (leezu) [inline scripts/bert/index.rst]: Usually we don't include pip install gluon-nlp line here.

Comment (jamiekang) [reply scripts/bert/index.rst]: I will remove the line and do PR again.

**Comment by mli** [comment]
Job PR-1159/4 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1159/4/index.html

**Comment by mli** [comment]
Job PR-1159/5 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1159/5/index.html

**Pull request merged**